### ISSUE TITLE
Sourcemap support in compiler

### ISSUE THREAD

**Issue by nojvek** [tdumitrescu/virtual-jade#21]
Currently when compiling with webpack, debugging an issue inside virtual-jade compilation is very hard.

The output is quite verbose. Source map support that maps to jade would be a time saver.

**Comment by tdumitrescu** [comment]
Good idea. If you're looking for inspiration for taking a stab at an implementation, there looks to be working [source map support in pugify](https://github.com/sidorares/pugify/blob/393b264e55e5ffc0867dec91f64f3ff55ce14b49/index.js#L83-L170).
